presto-common/src/main/java/com/facebook/presto/common/RuntimeStats.java

```diff
@@ -87,23 +89,33 @@ public Map<String, RuntimeMetric> getMetrics()
 
     public void addMetricValue(String name, long value)
```

Comment:
Can we remove this method and migrate all the calls to `addMetricValue `?
Keep this method just makes the system very prone to errors